src/components/Items/ProfileContact.js

```diff
@@ -71,15 +104,16 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     flex: 1,
-    paddingVertical: 12,
-    paddingHorizontal: 15
+    paddingVertical: dimen.normalizeDimen(12),
+    paddingHorizontal: dimen.normalizeDimen(15)
   },
-  fullname: {
+  fullname: (isMe) => ({
     fontSize: normalizeFontSize(14),
-    fontFamily: fonts.inter[500],
-    color: '#000',
-    lineHeight: normalizeFontSize(16.94)
-  },
+
+    color: isMe ? colors.darkBlue : 'black',
+    lineHeight: normalizeFontSize(16.94),
+    fontWeight: 'bold'
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated. The `paddingVertical` and `paddingHorizontal` values are now normalized using the `dimen.normalizeDimen` function. The `fullname` style now accepts a parameter `isMe` and changes the color based on its value. The `fontFamily` has been replaced with `fontWeight`. These changes seem to be necessary for the new functionality and there doesn't appear to be any issues with them.


<!-- This is an auto-generated comment by OSS CodeRabbit -->